hack/.structured_logging

```diff
@@ -1,2 +1,3 @@
 cmd/kubelet/...
+pkg/controller/serviceaccount/...
```

Comment:
I would not recommend marking this directory as migrated we it makes collecting approvals harder and we didn't get approval from component owner